src/main/java/mcp/mobius/waila/overlay/tooltiprenderers/TTRenderFluidBar.java

```diff
@@ -49,22 +56,39 @@ public TTRenderFluidBar() {
             formatNumber = GTUtility::formatNumbers;
         } else {
             bindColor = (fluidName) -> GL11.glColor4f(1F, 1F, 1F, 1F);
-            formatNumber = (number) -> String.format("%,d", number);
+            formatNumber = numberFormat::format;
         }
     }
 
     @Override
     public Dimension getSize(String[] params, IWailaCommonAccessor accessor) {
-        int commaCount = (int) (Math.floor((params[2].length() - 1) / 3D) + Math.floor((params[3].length() - 1) / 3D));
-        // ",".repeat(commaCount) doesn't exist in java 8 so do this instead.
-        StringBuilder sb = new StringBuilder(commaCount);
-        for (int i = 0; i < commaCount; i++) {
-            sb.append(",");
+        String barText;
+        if (!(params[0].equals("EMPTYFLUID") && params[1].equals("EMPTYFLUID"))) {
+            int commaCount = (int) (Math.floor((params[2].length() - 1) / 3D)
+                    + Math.floor((params[3].length() - 1) / 3D));
```

Comment:
wait, can't we just use `formatNumber(params[2])` & `formatNumber(params[3])` which will place commas automatically instead of calculating their amount?

btw NumberFormat doc says that grouping size is different depending on locale so current solution could lead to problems